test.sh

```diff
@@ -31,7 +32,7 @@ fi
 do_test() {
     for path in "${EXAMPLES_ARDUINO_UNO[@]}"
     do
-        "${ARDUINO_CLI}" compile --warnings all -v -b arduino:avr:uno "${path}"
```

Comment:
Why uno is replaced with mega?

Reply:
Because there are not enough  memory on uno board for using library with claiming,provisioning and other features.

Reply:
Is it possible to turn the features on/off depending on the version of the board?

Reply:
Yes, but we should decide - what features we should disable? I suggest to disable claiming and provisioning features.